components/forms/w-input.vue

```diff
@@ -1,19 +1,28 @@
 <template>
-  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation, aria }" :class="{ 'has-suffix': $slots.suffix, 'has-prefix': $slots.prefix }">
-    <div class="input" :class="inputWrapperClass">
+  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation, aria }">
+      <div :class="[inputWrapperClass, wrapperClass]">
       <slot name="prefix" :inputElement="inputEl" />
-      <input v-if="mask" v-bind="{ ...aria, ...$attrs, class: '' }" @blur="triggerValidation" ref="inputEl" :autocomplete="autocomplete" :id="id" :type="type">
-      <input v-else      v-bind="{ ...aria, ...$attrs, class: '' }" @blur="triggerValidation" ref="inputEl" :autocomplete="autocomplete" :id="id" :type="type" v-model="model">
+      <input v-if="mask" :class="inputClasses" v-bind="{ ...aria, ...$attrs, class: '' }" @blur="triggerValidation" ref="inputEl" :autocomplete="autocomplete" :id="id" :type="type">
+      <input v-else 
+      :class="[
+        inputClasses,
+        {
+          'pr-40': $slots.suffix,
```

Comment:
Do you think it makes sense to import these as e.g. `input.suffix`. and `input.prefix`?

Reply:
Sure, I fix.